services/motion/builtin/builtin_test.go

```diff
@@ -1701,8 +1750,11 @@ func TestMoveOnMapNew(t *testing.T) {
 		req := motion.MoveOnMapReq{
 			ComponentName: base.Named("test-base"),
 			Destination:   easyGoalInSLAMFrame,
-			SlamName:      slam.Named("test_slam"),
-			Extra:         map[string]interface{}{"smooth_iter": 0},
+			MotionCfg: &motion.MotionConfiguration{
```

Comment:
Can we please just change the source & destination to be more than 2600m away from each other so we don't need to specify the plan deviation meters?

Reply:
we need to specify a motion cfg here since our goal is within 1 m of the start